.github/workflows/ci.yml

```diff
@@ -781,24 +781,25 @@ jobs:
           change=$([ $current_size -gt $base_size ] && echo "increased❗" || ([ $current_size = $base_size ] && echo "not changed" || echo "decreased✅"))
           echo "change=${change}" >> $GITHUB_OUTPUT
           echo "diff=$(($current_size - $base_size))" >> $GITHUB_OUTPUT
-      - name: Find Comment
-        uses: peter-evans/find-comment@v2
-        id: fc
-        with:
-          issue-number: ${{ github.event.pull_request.number }}
-          comment-author: 'github-actions[bot]'
-          body-includes: '## ${{ matrix.app }} bundle size is'
-      - name: Create or update comment
-        uses: peter-evans/create-or-update-comment@v2
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Update PR Description
```

Comment:
@edwardlee-msft given this PR #5269 do we need to ensure any santization is happening if someone injects scripts into the PR description?

Reply:
Closing this comment thread for other sanitization-related ones